spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -926,6 +955,17 @@ private static java.sql.Date toDate(String value) {
               new StructType(dateStruct))
           .repartition(2);
 
+  private static final Dataset<Row> dateHourDF =
```

Comment:
I think dateDF is specifically to address date partition test, and dateHourDf doesnt indicate that is testing a case where hour is modeled as a string.  Maybe something descriptive like testPartitionTypeDF

Reply:
done. Had to rearrange date column order as spark insert was complaining about string to date conversion for non date column